src/UI/examples/Popover/show_popover_with_vertical_scrollbars.php

```diff
@@ -0,0 +1,32 @@
+<?php
+function show_popover_with_vertical_scrollbars()
+{
+	global $DIC;
+	$factory = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	$series = [
+		'Breaking Bad',
+		'Big Bang Theory',
+		'Dexter',
+		'Better Call Saul',
+		'Narcos',
+		'Ray Donovan',
+		'Simpsons',
+		'South Park',
+		'Fargo',
+		'Bloodline',
+		'The Walking Dead',
+		'New Girl',
+		'Sons of Anarchy',
+		'How I Met Your Mother',
+	];
+	$list = $renderer->render($factory->listing()->unordered($series));
+	$content = "<div style='max-height: 200px; overflow-y: auto; padding-right: 10px;'>{$list}</div>";
```

Comment:
@wanze the example is fine. All that is needed is a comment above describing, what is done here and that this is normally job of the (Less/CSS) of the respective renderer and MUST NEVER be done inline. As soon as we have components doing that, we can remove this example.